### PR TITLE
Unit9_hw

### PR DESCRIPTION
**Pull request by harumambura**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "yet_another_playwright",
+  "version": "1.0.0",
+  "description": "playwright test repo",
+  "main": "index.js",
+  "scripts": {},
```
Comment (4rtemOv) [inline package.json]: let's add scripts to launch the tests using one simple command

File: tests/my_test.spec.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+const { test, expect } = require("@playwright/test");
+
+test('should perform login', async({page}) => {
+    await page.goto("https://www.saucedemo.com/");
```
Comment (4rtemOv) [inline tests/my_test.spec.js]: let's add this url to the playwright config - baseUrl

File: tests/my_test.spec.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    await page.locator("#login-button").click();
+    await expect(page.locator(".title")).toContainText("Products");
+    await expect(page.getByText('Products')).toBeVisible();
+    await expect(page.locator('#shopping_cart_container a')).toBeVisible();
+    const products = await page.locator('.inventory_item').all();
+    expect(products.length).toBeGreaterThan(0);
+})
+
+test("should add product to the cart", async({page}) => {
+    await page.goto("https://www.saucedemo.com/");
```
Comment (4rtemOv) [inline tests/my_test.spec.js]: let's use beforeEach function to perform login before every test in this file

**Review by 4rtemOv — CHANGES_REQUESTED** [review]
Well done! We have some more time so let's make your code better using some additional functionality available in playwright

File: tests/my_test.spec.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+const { test, expect } = require("@playwright/test");
+const { beforeEach } = require("node:test");
```
Comment (AlinaZhykharieva) [inline tests/my_test.spec.js]: you are not using this import. beforeEach is the method of **test**. So can delete this line

**Comment by AlinaZhykharieva** [comment]
Really good job! Looks awesome! I`ve run your test. One suggestion from my side is to improve Readme (list scripts that you have in package.json). 
image

**Pull request merged**